diff --git a/utils/clone.py b/utils/clone.py
index 079c378c..41d2859c 100755
--- a/utils/clone.py
+++ b/utils/clone.py
@@ -126,7 +126,7 @@ def clone(args): # pylint: disable=too-many-branches, too-many-locals, too-many-
         cwd=gsupath,
         check=True)
     # Apply changes to gsutil
-    run(['git', 'apply'],
+    run(['git', 'apply', '--ignore-whitespace'],
         input=Path(__file__).with_name('gsutil.patch').read_text(encoding=ENCODING),
         cwd=gsupath,
         check=True,
